### DISCUSSION TITLE
Accessing 40-pin GPIO | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#261] [Q&A]
# Accessing 40-pin GPIO | Seeed Studio Wiki

ODYSSEY - X86J41x5

https://wiki.seeedstudio.com/ODYSSEY-X86J4105-GPIO/#interrupts#comments

**Comment by Winterian** [comment]
Hello

I've confirmed that the X86J4125 works good, but X86J4125 v2 has different coprocessor. (ATSAMD21 -> RP2040)
So, I would like to know the GPIO interrupt works in v2.

**Reply by MatthewJeffson** [reply]
Hi! This is the wiki of 40-pin GPIO and both versions share the same GPIO and should be working fine.
